gephi/gephi
# Make edge kind editable#2518

https://github.com/gephi/gephi/issues/2518

Edge kind is now editable from Datatable. 


How to test : 

Open Gephi with new Workspace
* Create 2 nodes
* Create a link with a type name "A"
* Select the 'Kind' on the Edges Table
* Change "A" to "Z"
* Notice the change is keep on table
* Check on overview , Appearance > Edges > Partition > Type   that now "Z" appears
